CHANGELOG.md

```diff
@@ -2,7 +2,13 @@
 
 All notable changes to this project will be documented in this file. This project adheres to [Semantic Versioning](http://semver.org/).
 
-## Unreleased
+## [0.0.6](https://github.com/rapid7/marionette.carpenter/releases/0.0.6) - 09/03/2024
+
+### Changed
+
+- Removed usage of `javascript:void(0)`
+
+## [0.0.?](https://github.com/rapid7/marionette.carpenter/releases/)
```

Comment:
what's up with the `?`

Reply:
There have been a few releases, and I am not sure what exactly has been included in them.
I could update the PR and copy whatever I can find in the actual releases if that's a better thing to do?

Reply:
@dwelch-r7 pushed the remaining releases in the CHANGELOG, making it complete.